{
  throw new UnsupportedOperationException("getUndo called on command that does returned false for isUndoable");
}
